graylog2-web-interface/src/components/indices/IndexSetFieldTypes/IndexSetCustomFieldTypeRemoveModal.test.tsx

```diff
@@ -85,7 +95,7 @@ describe('IndexSetFieldTypesList', () => {
       renderIndexSetCustomFieldTypeRemoveModal();
 
       const checkbox = await screen.findByText(/rotate affected indices after change/i);
-      const submit = await screen.findByTitle(/remove custom field type/i);
+      const submit = await screen.findByTitle(/remove field type overrides/i);
```

Comment:
Is it may possible to use `findByRole` here as well?

Reply:
@linuspahl `BootstrapModalForm` has a submit button inside which doesn't have a button role